src/custom/CatalogDetail/helper.ts

```diff
@@ -47,6 +47,20 @@ export const downloadFilter = (id: string, name: string): void => {
   linkElement.remove();
 };
 
+export const downloadPattern = (id: string, name: string, type: string): void => {
+  const pattern = type == 'design' ? 'patterns' : 'filters';
+  const dataUri = `${process.env.API_ENDPOINT_PREFIX}/api/content/${pattern}/download/${id}`;
```

Comment:
dataUri -> dataUrl

Reply:
@mdkaifansari04 how you are accessing env variable here directly in sistent?